src/components/slices/faculty-grid.module.scss

```diff
@@ -1,107 +1,171 @@
 @import '../../base/module';
+
 .FacultyGrid {
-    position: relative;
-    padding-top: 0;
+  position: relative;
+  padding-top: 0;
+  background-color: white;
 }
-.grid {
-    display: grid;
-    grid-template-columns: repeat(auto-fit, minmax(100%, 1fr));
-    column-gap: 30px;
-    row-gap: 60px;
 
-    @media (min-width: $tablet) {
-        grid-template-columns: repeat(auto-fit, minmax(400px, 1fr));
-    }
+.photoContainer {
+  display: flex;
+  gap: 30px;
+  flex-direction: column;
+  position: relative;
+  top: -165px;
```

Comment:
so the reason this is giving you trouble with the space below Is that absolute positioning maintains the spacing everywhere because it's out of the flow versus if you use margin-top: -165px etc which won't effect the spacing. It'll just take everything with it so then you can add proper spacing after. 

Do note that because container has a margin class declaration already you'll have to possibly be more specific with your css, like adding .Container ahead of .photoContainer. But it should work okay.